src/style.scss

```
@@ -465,3 +465,4 @@ $primary-color: #006eef;
     }
   }
 }
+@import '@wedevs/plugin-ui/styles.css';
```

Stylelint (17.3.0)

[error] Unexpected invalid position `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)